apps/omg_watcher_info/test/support/factory.ex

```diff
@@ -66,4 +73,49 @@ defmodule OMG.WatcherInfo.Factory do
       timestamp: sequence(:block_timestamp, fn seq -> seq * 1_000_000 end)
     }
   end
+
+  @doc """
+  Transaction factory.
+
+  Generates a transaction without any transaction output and no associated block.
+
+  To generate a transaction with closest data to production, consider associating the transaction
+  to a block and generating transaction outputs associated with this transaction.
+  """
+  def transaction_factory() do
+    %DB.Transaction{
+      txhash: sequence(:block_hash, fn seq -> <<seq::256>> end),
+      txindex: 0,
+      txbytes: :crypto.strong_rand_bytes(8),
```

Comment:
I'm against draining entropy for no reason, if you share my point alternative could be 
`0..255 |> Enum.shuffle() |> Enum.take(32) |> :erlang.list_to_binary()`

Reply:
Addressed in https://github.com/omisego/elixir-omg/pull/1224/commits/3649aa7c17b835428005deba8ad33e3ef85ab9c1